src/components/block-button.js

```diff
@@ -0,0 +1,31 @@
+window.blockUser = function(id) {
+  NAF.connection.adapter.publisher.handle.sendMessage({ kind: "block", whom: id });
```

Comment:
Hmm I think it's fine that we implement blocking as a proprietary thing in our adapter, but we shouldnt be reaching into internal datastructures of the adapter or be aware of janus protocol to do it, we should just expose a `block(clientId)` function on the adapter that does this internally.

Reply:
👍